.github/workflows/main.yml

```diff
@@ -122,7 +122,10 @@ jobs:
       - uses: actions/checkout@v4
         with:
           lfs: true
+          # Recursive checkout required to build Solidity contracts and bindings.
+          submodules: recursive
       - uses: ./.github/actions/rustup
+      - uses: risc0/foundry-toolchain@2fe7e70b520f62368a0e3c464f997df07ede420f
```

Comment:
Why is this needed for the `test` job? It should only be required for the `website-test` and `bonsai` jobs.

Reply:
yeah I'm also kinda surprised, but this is what CI gets without:
```
cargo test -F $FEATURE -F profiler --workspace --exclude doc-test
error: failed to run custom build command for `bonsai-ethereum-contracts v0.5.0 (/opt/actions-runner/_work/risc0/risc0/bonsai/ethereum)`
note: To improve backtraces for build dependencies, set the CARGO_PROFILE_TEST_BUILD_OVERRIDE_DEBUG=true environment variable to enable debug information generation.

Caused by:
  process didn't exit successfully: `/opt/actions-runner/_work/risc0/risc0/target/debug/build/bonsai-ethereum-contracts-560e815577b67762/build-script-build` (exit status: 1)
  --- stderr
  Error: 
  Failed to resolve file: "/opt/actions-runner/_work/risc0/risc0/bonsai/ethereum/lib/forge-std/src/Test.sol": No such file or directory (os error 2).
   Check configured remappings..
      --> "/opt/actions-runner/_work/risc0/risc0/bonsai/ethereum/contracts/BonsaiCheats.sol"
          "forge-std/Test.sol"
  Error: `forge build` exited with failed status: exit status: 1
```